benchmarks/2019.json

```diff
@@ -4445,24 +4517,6 @@
       0
     ]
   },
-  {
```

Comment:
Past: 
```
{"level":"debug","message":"measure 438 2017 submission method electronicHealthRecord single_performance_rate_measurements: 0 measurements","timestamp":"2019-01-04T00:56:44.940Z"}
{"level":"debug","message":"measure 438 2017 submission method electronicHealthRecord single_performance_rate_measurements: 0 measurement-stratas","timestamp":"2019-01-04T00:56:44.940Z"}
{"level":"debug","message":"measure 438 has only 0 qualified TIN-NPIs reporting for submission method electronicHealthRecord, so we skip it","timestamp":"2019-01-04T00:56:44.940Z"}
```

Present:
```
{"level":"debug","message":"measure 438 2017 submission method electronicHealthRecord single_performance_rate_measurements: 0 measurements","timestamp":"2019-09-05T22:53:16.388Z"}
{"level":"debug","message":"measure 438 2017 submission method electronicHealthRecord single_performance_rate_measurements: 0 measurement-stratas","timestamp":"2019-09-05T22:53:16.388Z"}
{"level":"debug","message":"measure 438 has only 0 qualified TIN-NPIs reporting for submission method electronicHealthRecord, so we skip it","timestamp":"2019-09-05T22:53:16.389Z"}
```
Need to see where this was added and if we need to add back manually

Reply:
Was added here:
https://github.com/CMSgov/qpp-measures-data/pull/234/files#diff-19cc2b2b0a838f96141210f2b77688c9R3199

Reply:
This was also added in `2019 benchmarks update MAR3`
```

Statin Therapy for the Prevention and Treatment of Cardiovascular Disease | 438 | Registry/QCDR | Process | Y | 78.72 - 91.66 | 91.67 - 95.64 | 95.65 - 97.37 | 97.38 - 99.99 | -- | -- | -- | 100 | Yes
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --



```
keep or remove?

Reply:
this should both be removed. and unfortunately were still in the file we'd been using to update.